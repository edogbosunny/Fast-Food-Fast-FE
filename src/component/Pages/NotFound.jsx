import React from 'react';
import '../../styles/style.css';

const NotFound = () => (
  <div className='container not-found'>
    <h3>Not Found 404</h3>
    <div>
      <a to='/'>
        <div>..go back Home....</div>
      </a>
    </div>
  </div>
);

export default NotFound;
