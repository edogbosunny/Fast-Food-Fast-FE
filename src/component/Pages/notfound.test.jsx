import { shallow } from 'enzyme';
import React from 'react';
import NotFound from './NotFound';

describe('Login Container', () => {
  const wrapper = shallow(
    <NotFound />,
  );

  it('should mount the login page Yeah!', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
