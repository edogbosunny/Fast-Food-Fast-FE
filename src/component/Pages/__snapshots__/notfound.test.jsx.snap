// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Container should mount the login page Yeah! 1`] = `
<div
  className="container not-found"
>
  <h3>
    Not Found 404
  </h3>
  <div>
    <a
      to="/"
    >
      <div>
        ..go back Home....
      </div>
    </a>
  </div>
</div>
`;
